content/zh/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -0,0 +1,451 @@
+---
+title: 众所周知的标签、注解和污点
```

Comment:
I would consider leaving “label”, “annotation” and “taint” untranslated.
Alternatively, make the English terms clear in the page introduction.